tpke/benches/benchmarks.rs

```diff
@@ -143,5 +148,50 @@ pub fn bench_decryption(c: &mut Criterion) {
     }
 }
 
-criterion_group!(benches, bench_decryption);
+pub fn bench_random_poly(c: &mut Criterion) {
+    use rand::SeedableRng;
+    let mut group = c.benchmark_group("RandomPoly");
+    type E = ark_bls12_381::Bls12_381;
+    group.sample_size(10);
+
+    for threshold in [20, 40, 60, 80, 100] {
```

Comment:
```suggestion
    for threshold in [1, 2, 4, 8, 16, 32, 64] {
```
Nothing wrong with the values you've selected, but this suggestion is closer to the real-life parameters our users are going to use.